docs/_toc.yml

```diff
@@ -0,0 +1,8 @@
+format: jb-article
+root: index
+sections:
+- file: plugins/index
+  sections:
+  - file: plugins/guides
+  - file: plugins/manifest
+  - file: plugins/contributions
```

Comment:
Right now I'm getting warnings about documents not begin in the toc tree. I think this results in missing links. Here's the relevant output I get from `jb docs build`:

```
checking consistency... /Users/nclack/src/CZI/npe2/docs/plugins/_layer_data_guide.md: WARNING: document isn't included in any toctree
/Users/nclack/src/CZI/npe2/docs/plugins/_readers_guide.md: WARNING: document isn't included in any toctree
/Users/nclack/src/CZI/npe2/docs/plugins/_sample_data_guide.md: WARNING: document isn't included in any toctree
/Users/nclack/src/CZI/npe2/docs/plugins/_widgets_guide.md: WARNING: document isn't included in any toctree
/Users/nclack/src/CZI/npe2/docs/plugins/_writers_guide.md: WARNING: document isn't included in any toctree
/Users/nclack/src/CZI/npe2/docs/plugins/first_plugin.md: WARNING: document isn't included in any toctree
done
preparing documents... done
writing output... [100%] plugins/manifest                                                                                               
/Users/nclack/src/CZI/npe2/docs/plugins/_readers_guide.md:14: WARNING: 'myst' reference target not found: 
/Users/nclack/src/CZI/npe2/docs/plugins/_readers_guide.md:3: WARNING: 'myst' reference target not found: 
/Users/nclack/src/CZI/npe2/docs/plugins/_writers_guide.md:3: WARNING: 'myst' reference target not found: 
/Users/nclack/src/CZI/npe2/docs/plugins/contributions.md:3: WARNING: 'myst' reference target not found: 
/Users/nclack/src/CZI/npe2/docs/plugins/guides.md:23: WARNING: 'myst' reference target not found: 
/Users/nclack/src/CZI/npe2/docs/plugins/guides.md:3: WARNING: 'myst' reference target not found: 
/Users/nclack/src/CZI/npe2/docs/plugins/guides.md:3: WARNING: 'myst' reference target not found: 
/Users/nclack/src/CZI/npe2/docs/plugins/index.md:10: WARNING: 'myst' reference target not found: ./best-practices
/Users/nclack/src/CZI/npe2/docs/plugins/index.md:12: WARNING: 'myst' reference target not found: find-and-install-plugin
/Users/nclack/src/CZI/npe2/docs/plugins/index.md:3: WARNING: 'myst' reference target not found: npe2-migration-guide
/Users/nclack/src/CZI/npe2/docs/plugins/index.md:6: WARNING: undefined label: plugins-for-plugin-developers
/Users/nclack/src/CZI/npe2/docs/plugins/index.md:6: WARNING: undefined label: hook-specifications-reference
/Users/nclack/src/CZI/npe2/docs/plugins/index.md:45: WARNING: undefined label: best-practices
```

Reply:
> Right now I'm getting warnings about documents not begin in the toc tree. I think this results in missing links.

hmm, I think jupyter book might be a little aggressive with the warnings... (i had an issue about other warnings somewhere over there and they made it sound like the warning was in error, not the code).  I'm pretty sure the top warnings (`document isn't included in any toctree`) are fine.  We don't _want_ those to be in the toctree, they are all used for includes...  

Were you able to find a link that doesn't work?  They all seem to work for me.  (for instance, go to a contribution and click on the corresponding "Guide" link)

Reply:
I thought the "build run and publish" link on the plugin's index page was wrong and the link to "contributions" in the "Fields" section of the manifest guide looked wrong.

Reply:
> "build run and publish" link on the plugin's index page

that's an internal page link, works fine...

> the link to "contributions" in the "Fields" section

yep, that one was wrong (fixed, thanks), but just needed fixing in the source code itself (i.e. the actual link was wrong).  unrelated to the jb warnings.